locales/nl.json

```diff
@@ -1,315 +1,315 @@
+    "Sort videos by: ": "Video's sorteren op: ",
+    "published": "publicatiedatum",
+    "published - reverse": "publicatiedatum - omgekeerd",
     "alphabetically": "alfabetische volgorde",
-    "alphabetically - reverse": "alfabetisch - omgekeerd",
-    "channel name": "kanaal naam",
-    "channel name - reverse": "kanaal naam - omgekeerd",
-    "Only show latest video from channel: ": "Laat alleen laatste video van kanaal zien: ",
-    "Only show latest unwatched video from channel: ": "Laat alleen de laatste onbekeken video zien van kanaal: ",
-    "Only show unwatched: ": "Laat alleen onbekeken videos zien: ",
-    "Only show notifications (if there are any): ": "Laat alleen notificaties zien (als die er zijn): ",
-    "Data preferences": "Gegevens voorkeuren",
+    "alphabetically - reverse": "alfabetische volgorde - omgekeerd",
+    "channel name": "kanaalnaam",
+    "channel name - reverse": "kanaalnaam - omgekeerd",
+    "Only show latest video from channel: ": "Alleen nieuwste video van kanaal tonen: ",
+    "Only show latest unwatched video from channel: ": "Alleen nieuwste niet-bekeken video van kanaal tonen: ",
+    "Only show unwatched: ": "Alleen niet-bekeken videos tonen: ",
+    "Only show notifications (if there are any): ": "Alleen meldingen tonen (als die er zijn): ",
+    "Data preferences": "Gegevensinstellingen",
     "Clear watch history": "Kijkgeschiedenis wissen",
-    "Import/export data": "Importeer/Exporteer gegevens",
-    "Change password": "Wijzig wachtwoord",
-    "Manage subscriptions": "Abonnees beheren",
-    "Manage tokens": "Munten beheren",
+    "Import/export data": "Gegevens im-/exporteren",
+    "Change password": "Wachtwoord wijzigen",
+    "Manage subscriptions": "Abonnementen beheren",
+    "Manage tokens": "Toegangssleutels beheren",
     "Watch history": "Kijkgeschiedenis",
     "Delete account": "Account verwijderen",
-    "Administrator preferences": "Voorkeuren van de beheerder",
-    "Default homepage: ": "Standaard homepage: ",
-    "Feed menu: ": "",
-    "Top enabled? ": "",
-    "CAPTCHA enabled? ": "",
-    "Login enabled? ": "",
-    "Registration enabled? ": "",
-    "Report statistics? ": "",
-    "Save preferences": "Opslaan voorkeuren",
-    "Subscription manager": "Abonnees beheerder",
-    "Token manager": "",
-    "Token": "",
-    "`x` subscriptions": "`x` abonnees",
-    "`x` tokens": "",
-    "Import/export": "Importeer/Exporteer",
-    "unsubscribe": "abonnement opzeggen",
-    "revoke": "",
-    "Subscriptions": "Abonnees",
-    "`x` unseen notifications": "`x` onbekeken notificaties",
+    "Administrator preferences": "Beheerdersinstellingen",
+    "Default homepage: ": "Standaard startpagina: ",
+    "Feed menu: ": "Feedmenu",
+    "Top enabled? ": "Bovenkant inschakelen",
+    "CAPTCHA enabled? ": "CAPTCHA gebruiken?",
+    "Login enabled? ": "Inloggen toestaan?",
+    "Registration enabled? ": "Registratie toestaan?",
+    "Report statistics? ": "Statistieken bijhouden?",
```

Comment:
Likely a couple elsewhere I missed, but these 4 appear to be missing the trailing space, should be `Statistieken bijhouden? `, `Registratie toestaan? `, etc.

Reply:
You're right, I did not expect spaces there, so I missed them. Will fix that too.

Reply:
Also fixed now :)